### DISCUSSION TITLE
How long does it take to audit a github sponsorship？

### DISCUSSION THREAD

**Discussion by yu199195** [community/community#19014] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi  github community,

I have submitted my sponsorship information for a long time, and now it is always pedding status, how long does it take?

**Comment by melvincwng** [comment, marked as answer]
Hi @yu199195,

Based on what I could find, it generally takes a few days to weeks for approval regarding GitHub sponsorship page. From my own experience, it took a few days before my sponsorship application was approved.

But since for you, you've mentioned that your application has not been approved or processed for a long time, my advice would be to submit a ticket on [GitHub Support](https://support.github.com/) for the support engineers to investigate the issue.

In the meantime, the only option (other than submitting the ticket) would be to just wait patiently till your sponsorship application is approved. 

Hope this helps. If this answer has helped you, please kindly 'Mark as Answer' so that others know this issue is already addressed.

Warm Regards
Melvin

Reference:
1. https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account
2. https://github.community/t/github-sponsors-pending-for-approval/125778
3. https://support.github.com/

**Reply by yu199195** [reply]
Marked as answer

**Comment by yu199195** [comment]
@melvincwng  thank you very much

**Reply by melvincwng** [reply]
@yu199195, no problem, glad that I could help :)

Please kindly 'Mark as answer' so that others know this issue is resolved/closed.

Thank you.
